docs/src/intro-java.md

```diff
@@ -131,7 +131,7 @@ By default browsers launched with Playwright run headless, meaning no browser UI
 - Java 8 or higher.
 - Windows 10+, Windows Server 2016+ or Windows Subsystem for Linux (WSL).
 - MacOS 12 Monterey, MacOS 13 Ventura, or MacOS 14 Sonoma.
-- Debian 11, Debian 12, Ubuntu 20.04 or Ubuntu 22.04.
+- Debian 11, Debian 12, Ubuntu 20.04 or Ubuntu 22.04, Ubuntu 24.04 - (x86-64 or arm64 architecture).
```

Comment:
`, on x86-64 and arm64 architecture.` is more clear that it relates to all versions rather than just u24.